docs/content/en/docs/yaml-crd-ref/_index.md

```diff
@@ -1,28 +1,22 @@
 ---
-title: CRD YAML Reference
-description: Reference pages for the YAML files that define the KLT CRDs
+title: Manifest CRD Reference
```

Comment:
In my opinion we could rename this to just `CRD Reference` - what do you think?

Reply:
I agree with this :)

Reply:
I happily defer to you guys!